exchange/exchangetest/append-bidder-names.json

```diff
@@ -137,6 +138,7 @@
               "crid": "creative-3",
               "ext": {
                 "prebid": {
+                  "bidid": "mock_uuid",
                   "targeting": {
                     "hb_bidder": "appnexus",
                     "hb_bidder_appnexus": "appnexus",
```

Comment:
Thoughts on if this should be disabled by default on the exchange tests to avoid changing all of them?

Reply:
Default is false. Reason why it's in all tests is because inserting bidID logic is placed directly in HoldAction. So I modified `newExchangeForTests` function and hardcoded mock bidId generator to true and this applies to all tests in exchange test. This unit test is the only one good place to test it.

Reply:
Added separate unit test for this.

Reply:
Also reverted back json files and added new for new tests